manifest.md

```diff
@@ -34,13 +34,9 @@ Unlike the [image index](image-index.md), which contains information about a set
     - **`mediaType`** *string*
 
         This [descriptor property](descriptor.md#properties) has additional restrictions for `config`.
-        Implementations MUST support at least the following media types:
-
-        - [`application/vnd.oci.image.config.v1+json`](config.md)
+        Implementations MUST support any media type that complies with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
```

Comment:
I am a bit confused by this change - it now mandates implementations to support _any_ media types, which isn't feasible.

Would it be clearer for you if we added a single line on line 40 like this:
```
 Implementations MUST support at least the following media types:
        - [`application/vnd.oci.image.config.v1+json`](config.md)
 
        Manifests concerned with portability SHOULD use one of the above media types.

        **Manifests MAY use, and implementations MAY support, other media types not defined here.**

        If the manifest uses a different media type than the above, it MUST comply with [RFC 6838][rfc6838], including the [naming requirements in its section 4.2][rfc6838-s4.2], and MAY be registered with [IANA][iana].
```

Reply:
I suppose it depends what kind of implementation you are, a registry can trivially support any media type, it just has to store the blob. A container runtime needs the information in there, so it can only support known media types, or maybe also their ducks.